luigi-pipeline/main.py

```diff
@@ -233,11 +233,11 @@ def program_args(self):
         report_name = 'report_pmc_test_%s.html' % time.strftime("%Y%m%d-%H%M%S")
 
         # Build validation command. No connection has to be made to the database or web server.
-        docker_command = 'docker run --rm -v %s:/study/ -v %s:/cbioportal_db_info/ -v %s:/html_reports/ %s' \
+        docker_command = 'docker run --rm -v /staging/cbioportal-pmc/config/portal.test-staging.properties:/cbioportal/portal.properties -v %s:/study/ -v %s:/cbioportal_db_info/ -v %s:/html_reports/ %s' \
```

Comment:
Please replace the `/staging/cbioportal-pmc/config/portal.test-staging.properties` part with either a configurable path (like the transmart-copy path) or a path relative to, e.g., the config dir.

Reply:
is this related to the TMT-1057 ticket? should I wait for that to be completed first? @ewelinagr maybe you can comment on this

Reply:
It's related, but that one is for generating the properties file. This is about how you refer to the file location in the pipeline code.